core/types/transaction_test.go

```diff
@@ -281,7 +281,7 @@ func testTransactionPriceNonceSort(t *testing.T, baseFee *big.Int) {
 	signer := LatestSignerForChainID(common.Big1)
 
 	// Generate a batch of transactions with overlapping values, but shifted nonces
-	groups := map[common.Address]Transactions{}
+	groups := map[common.Address][]*Transaction{}
```

Comment:
Why did you make these changes?

Reply:
The usage of types.Transactions was a bad ide / misguided. The reason that type was introduced is to have a derive sha method. But we started yoloing the code and using it as a shorthand for []*types.Transaction, which was not the goal. It's bad design to have 2 things mean the same thing in the code, hence why I try to be canonical and use the slice version.